src/layered_config_tree/main.py

```diff
@@ -365,18 +370,31 @@ def get(self, keys: str | list[str], default_value: Any = None) -> Any:
 
         Returns
         -------
-        The value at the key or nested keys. If the key path exists except for the
-        final value, the ``default_value`` is returned.
+        The value at the key or nested keys at the requested layer. ``default_value``
+        is returned if
+        - the key path exists except for the final value.
+        - the value does not exist at the requested layer.
         """
         if not isinstance(keys, (str, list)):
             raise TypeError("The 'keys' parameter must be a string or a list of strings.")
 
         if isinstance(keys, str):
-            return self[keys] if keys in self._children else default_value
+            if keys not in self._children:
+                return default_value
+            child = self._children[keys]
+            if isinstance(child, ConfigNode):
+                child_layers = [metadata["layer"] for metadata in child.metadata]
```

Comment:
Can't this be simplified to be simply `child.get_value(layer=layer, default_value=default_value)` if we add that argument to `ConfigNode`?

It's a shame that we wouldn't be able to be consistent with the argument order of `layer` and `default_value` between the `LayeredConfigTree` and `ConfigNode` methods if we maintain backwards compatibility.

Reply:
I added it but the problem is this would necessitate a major release...and then we'd have to major release _again_ when we remove `get_from_layer`